port_ocean/clients/port/client.py

```diff
@@ -75,3 +76,16 @@ async def get_org_id(self) -> str:
         handle_status_code(response)
 
         return response.json()["organization"]["id"]
+
+    async def update_integration_state(self, state: dict[str, Any]) -> dict[str, Any]:
+        logger.debug(f"Updating integration state with: {state}")
+        response = await self.client.patch(
+            f"{self.api_url}/integration/{self.integration_identifier}/state",
+            headers=await self.auth.headers(),
+            json=state,
+        )
+        handle_status_code(response, should_raise=False, should_log=True)
+        if not response.is_error:
+            logger.info("Integration state updated successfully")
```

Comment:
and use the `should_log` here as well

Reply:
WDYM?

Reply:
once you add the `should_raise` and `should_log` then decide if to log based on the parameter passed